core-bundle/src/Resources/contao/library/Contao/Template.php

```diff
@@ -157,6 +157,11 @@ public function __get($strKey)
 			return $this->arrData[$strKey];
 		}
 
+		if ($strKey === 'requestToken' && !\array_key_exists($strKey, $this->arrData))
```

Comment:
Could we alternatively do the following in the constructor and leave the magic get/isset function as is?
```php
$this->arrData['requestToken'] = static fn() => htmlspecialchars(System::getContainer()->get('contao.csrf.token_manager')->getDefaultTokenValue());
```